src/sympc/module/nn/linear.py

```diff
@@ -0,0 +1,82 @@
+# stdlib
+from collections import OrderedDict
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Tuple
+
+# third party
+import torch
+
+from sympc.tensor import MPCTensor
+
+RTOL = 10e-4
+
+
+class Linear:
+    __slots__ = ["weight", "bias", "session", "in_features", "out_features"]
+
+    def __init__(self, session) -> None:
```

Comment:
So for the moment this class can only be used through a SyNet. Nothing to change, but we need to remember this.

Reply:
Yep